api/src/main/java/org/openmrs/api/impl/PersonServiceImpl.java

```diff
@@ -485,19 +485,10 @@ public Relationship saveRelationship(Relationship relationship) throws APIExcept
 	 */
 	@Override
 	public Relationship voidRelationship(Relationship relationship, String voidReason) throws APIException {
```

Comment:
`voidReason` is no longer being used. Maybe you can remove this parameter.
```suggestion
	public Relationship voidRelationship(@Nonnull final Relationship relationship) throws APIException {
```